3party/opening_hours/opening_hours_integration_tests/opening_hours_integration_tests.cpp

```diff
@@ -100,13 +94,13 @@ enum
 };
 using TRuleFeatures = std::array<bool, Count_>;
 
-std::ostream & operator<<(std::ostream & ost, TRuleFeatures const & f)
+inline std::ostream & operator<<(std::ostream & ost, TRuleFeatures const & f)
 {
  std::copy(begin(f), end(f), std::ostream_iterator<bool>(ost, "\t"));
  return ost;
 }
 
-TRuleFeatures DescribeRule(osmoh::TRuleSequences const & rule)
+inline TRuleFeatures DescribeRule(osmoh::TRuleSequences const & rule)
```

Comment:
These methods are not one-liners, do we really need to mark them as inline? Note that inline annotation is almost archaic as modern compilers (at least clang) are smart enough to recognise whether is possible to inline a function or not.  IMHO, the main usage of inline annotation is to define functions in header files.